### DISCUSSION TITLE
How to get the player back on the lock screen?

### DISCUSSION THREAD

**Discussion by littlesandra88** [PaulWoitaschek/Voice#2256] [Q&A] [answered]
Dear all =)

When I installed Voice it had the player on the lock screen. Somehow I got this removed.

Does anyone know how I can the player back on the lock screen?

Hugs,
Sandra =)

**Comment by PaulWoitaschek** [comment]
Hey Sandra. That sounds like you updated you Android version. Could that be the case?

**Reply by littlesandra88** [reply]
No, I think I long tapped on it, and then selected "hide" or something similar to that. And now I can't get it back =)

**Reply by PaulWoitaschek** [reply, marked as answer]
Hmm when maybe long press the app icons, go to settings and enable the notifications again

**Reply by littlesandra88** [reply]
Thanks so much! :) That did it.

It is such a wonderful player. I can't thank you enough for it. :)
